README.md

```diff
@@ -38,7 +38,7 @@ Sometimes you'll have an activity that should handle several kinds of deep links
 `@DeepLinks` annotation to register multiple deep links on an activity:
 
 ```java
-@DeepLinks({"example.com/deepLink/{id}", "example.com/anotherDeepLink"})
+@DeepLinks({"foo://example.com/deepLink/{id}", "foo://example.com/anotherDeepLink"})
```

Comment:
If this is a breaking change, is it worth throwing an exception for a missing scheme in the processor?